src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -83,21 +83,25 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
       onEventUpdated({
         ...shapeToolData,
         selectedEstimate: { iri: snsPredictedIri, value: snsPredictedFailureRate },
-        probability: snsPredictedFailureRate,
+        probability: Number(snsPredictedFailureRate),
       });
     }
     if (event.target.value === RadioButtonType.Operational) {
       // Updated when we switch to Oper. rate:
       onEventUpdated({
         ...shapeToolData,
         selectedEstimate: { iri: snsOperationalIri, value: snsOperationalFailureRate },
-        probability: snsOperationalFailureRate,
+        probability: Number(snsOperationalFailureRate),
```

Comment:
If possible, it would be better to give the type to `snsOperationalFailureRate` than casting it.

Reply:
Fixed.